### DISCUSSION TITLE
Hypergeometry Support

### DISCUSSION THREAD

**Discussion by ericcberg** [deltagreen-foundryvtt/delta-green-foundry-vtt-system#90] [Ideas]
Are there any plans to add being able to add Tome and Ritual objects and adding them to character sheets?

**Comment by TheLastScrub** [comment]
Currently these have just been put into the 'other gear' section. For the most part I've been fine with that, since they don't really come up that much in any given session for me. I usually only see a ritual activation once every couple of sessions at the most.

But recently Arc Dream did ask us to look into moving ritual activation out of the "skills" section (since it isn't really a skill per se). Not entirely sure when we'll swing around to this, but @jalensailin and I did briefly discuss as part of that change, whether it would make sense to move some of that into its own 'ritual' section. So if/when we do that change, I think adding a 'Ritual' type object and its own section might make sense.

I am not sure I'd make a separate Item type just for tomes though. Is there any mechanical significance to them beyond the possibility of allowing for learning a ritual? Or is the request more just organizational in nature?

**Comment by ericcberg** [comment]
Mostly an organizational tool (so you could add Ritual objects to it) and
as a way to note the stats for it.

On Sun, Oct 1, 2023 at 6:34 PM TheLastScrub ***@***.***>
wrote:

> Currently these have just been put into the 'other gear' section. For the
> most part I've been fine with that, since they don't really come up that
> much in any given session for me. I usually only see a ritual activation
> once every couple of sessions at the most.
>
> But recently Arc Dream did ask us to look into moving ritual activation
> out of the "skills" section (since it isn't really a skill per se). Not
> entirely sure when we'll swing around to this, but @jalensailin
> <https://github.com/jalensailin> and I did briefly discuss as part of
> that change, whether it would make sense to move some of that into its own
> 'ritual' section. So if/when we do that change, I think adding a 'Ritual'
> type object and its own section might make sense.
>
> I am not sure I'd make a separate Item type just for tomes though. Is
> there any mechanical significance to them beyond the possibility of
> allowing for learning a ritual? Or is the request more just organizational
> in nature?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/TheLastScrub/delta-green-foundry-vtt-system-unofficial/discussions/90#discussioncomment-7159889>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADCQVWLWZ2LA6ENVCPFG2CDX5HVXHANCNFSM6AAAAAA5OSRVDY>
> .
> You are receiving this because you authored the thread.Message ID:
> <TheLastScrub/delta-green-foundry-vtt-system-unofficial/repo-discussions/90/comments/7159889
> @github.com>
>


-- 
Eric Christian Berg
